azure-stack/hci/concepts/security-features.md

```diff
@@ -0,0 +1,132 @@
+
+All data-at-rest on your Azure Stack HCI cluster is protected with BitLocker XTS-AES 256-bit encryption. When you deploy your cluster, you have the option to modify security settings. By default, data-at-rest encryption is enabled on data volumes created during deployment. We recommend that you accept the default setting.
+
+We recommend that you store BitLocker recovery keys in a secure location outside of the system. Once Azure Stack HCI is successfully deployed, you can retrieve BitLocker recovery keys.
+
+For more information about BitLocker, see:
+
+- [Use BitLocker with Cluster Shared Volumes (CSV)](../manage/bitlocker-on-csv.md).
+- [Enable Bitlocker encryption on newly created volumes](#manage-bitlocker-encryption).
+- [BitLocker encryption on Azure Stack HCI (preview)](./security-bitlocker.md).
+
+## Local built-in user accounts
+
+The names of local built-in users associated with the `RID 500` and `RID 501` accounts have been updated:
+
+|Name |Enabled |Description |
+|-----|-----|-----|
+|ASBuiltInAdmin |True |Built-in account for administering the computer/domain |
+|ASBuiltInGuest |False |Built-in account for guest access to the computer/domain |
+
+> [!Important]
+> We recommend that you create your own local administrator account, and that you disable the well-known `RID 500` user account.
+
+The `ASBuiltInGuest` account is protected by the security baseline drift control mechanism. For more information, see [Security baseline settings for Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+
+We also added two new components in the 23H2 release:
+
+- [Secrets management engine](../whats-new.md).
+- [SIEM syslog forwarder agent](../whats-new.md).
+
+## Manage secrets in Azure Stack HCI
+
+The Lifecycle Manager (LCM) stack requires multiple components to maintain secure communications with other infrastructure resources and services. To ensure security, we implemented internal secret creation and rotation capabilities.
+
+All services exposed internally by LCM have authentication or encryption certificates associated with them. When you look into your cluster nodes, you will see several certificates created under the path `LocalMachine/Personal certificate store (Cert:\LocalMachine\My).
+
+This new component enables the following capabilities:
+
+- The ability to create certificates during deployment and post deployment cluster scale actions.
+- Automated auto-rotation mechanism before certificates expire, and a customer option to rotate certificates during the lifetime of the cluster.
+- The ability to monitor and alert if certificates are still valid.
+
+> [!NOTE]
+> This action will take place using LCM actions and will last about 10 minutes, depending on the size of the cluster.  This action affects all nodes that use LCM, and requires the user to belong to the `LCM authorization` group (PREFIX-ECESG) and `CredSSP` for remote Powershell invocation or remote desktop protocol (RDP) connection.
+
```

Comment:
Change: This action affects all nodes that use LCM
Seems to imply that there are nodes that do not LCM. Hence, confusing. Confirm w/ Carlos and rewrite.

Reply:
@carlosmayol - Can we clarify the language in the note above to avoid customer confusion? 

Specifically, can we revise to:

Existing: This action affects all nodes that use LCM, and requires the user to belong...
Revised: This action requires the user to belong...

Thx.